addons/test_discuss_full/tests/test_performance.py

```diff
@@ -99,7 +99,7 @@ def test_init_messaging(self):
         self.maxDiff = None
         self.users[0].flush()
         self.users[0].invalidate_cache()
-        with self.assertQueryCount(emp=90):
+        with self.assertQueryCount(emp=108):
```

Comment:
This seems too much, it should compute the unread counter for all channel_partner at once.